### PR TITLE
python-psutil 5.9.5 (new formula) aws2-wrap: update build

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/p/python-psutil.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class PythonPsutil < Formula
+  desc "Cross-platform lib for process and system monitoring in Python"
+  homepage "https://github.com/giampaolo/psutil"
+  url "https://files.pythonhosted.org/packages/d6/0f/96b7309212a926c1448366e9ce69b081ea79d63265bde33f11cc9cfc2c07/psutil-5.9.5.tar.gz"
+  sha256 "5410638e4df39c54d957fc51ce03048acd8e6d60abc0f5107af51e5fb566eb3c"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.10" => [:build, :test]
```
Comment (fxcoudert) [inline Formula/p/python-psutil.rb:8]: Unless we have actual uses for it in old python versions, I'd say we keep it to current version.

Comment (chenrui333) [reply Formula/p/python-psutil.rb:8]: make sense, let me switch to use the latest version.

**Review by fxcoudert — APPROVED** [review]

**Comment by chenrui333** [comment]
`  ModuleNotFoundError: No module named 'setuptools'`

**Review by p-linnane — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks folks, merging now.

**Pull request merged**